girder_monkeybrains/__init__.py

```diff
@@ -41,9 +44,8 @@ def setCollectionMonkeybrains(self, collection, params):
             del collection[MONKEYBRAINS_FIELD]
         else:
             collection[MONKEYBRAINS_FIELD] = monkeybrains
-        self.model('collection').save(collection, validate=False)
-        return self.model('collection').filter(collection,
-                                               self.getCurrentUser())
+        ModelImporter.model('collection').save(collection, validate=False)
+        return ModelImporter.model('collection').filter(collection, self.getCurrentUser())
```

Comment:
We just import these models statically now, i.e.

```python
from girder.models.collection import Collection


Collection().save(...)
```

Also the call to `filter` should be replace with the `filtermodel` decorator.

Reply:
Normatively: if the model you want to load is known statically, use the static instantiation.